site/using-weave/weave-cloud.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Using Weave Cloud to get started with Weave Net
+menu_order: 10
+---
+
+Weave Net provides a simple to deploy networking solution for containerized apps. Here, we describe how to manage a Weave container network using a sample application which consists of two simple `netcat` services deployed to containers on two separate hosts.
+
+This section contains the following topics:
+
+ * [Signing in to Weave Cloud](#signing-in)
+ * [Launching Weave Net](#launching)
+ * [Launching on Additional Hosts](#peer-connections)
+ * [How Peer Discovery Works](#how-it-works)
+
+### <a name="signing-in"></a>Signing in to Weave Cloud
+
+Weave Cloud is a fast and simple way to visualize, manage and monitor
+containers and microservices.  In order to use Weave Cloud to set up
+Weave Net, you must sign up to Weave Cloud and create an "Instance".
+
+The instance will have a Service Token which is a 32-character string
+used to uniquely identify it.  Get hold of the Service Token for your
+Instance.
+
```

Comment:
I've been calling this a cloud token, but I see it is service token in the UI. 

Could use my screen capture here (https://github.com/weaveworks/guides/blob/master/images/weave-cloud-token.png)